migrations/2018-10-15-141132_seedfile/up.sql

```diff
@@ -0,0 +1,9 @@
+INSERT INTO projects (title, name, status, link, email, content)
+VALUES ('Whimsy', 'bwinton', 'in-progress', 'https://addons.mozilla.org/en-US/firefox/addon/whimsycorn/', 'someone@mozilla.com', 'Whimsy is always in progress');
+
+INSERT INTO projects (title, name, status, link, email, content)
+VALUES ('Doh', 'jduell', 'done', 'https://bugzilla.mozilla.org/show_bug.cgi?id=1495583', 'someone@mozilla.com', 'Lets make the doh options page in about:preferences look better and actually give some information to the user.');
```

Comment:
Maybe put a capital `H` here, so that it looks less like
![doh](https://media0.giphy.com/media/l0G18G3m69vQCOddm/200.gif?cid=3640f6095bc4aac8355767676b4c2ddc)